### PR TITLE
[risk=no][no ticket]Allow verbosity to be adjusted via environment variable

### PR DESCRIPTION
**Pull request by dmohs**
Not part of main codebase.

### CODE REVIEW

**Review by aweng98 — APPROVED** [review]

**Comment by jmthibault79** [comment]
linter doesn't like it.
```
$ yarn clean && tsc -p tsconfig.json
$ rimraf tsc-out
libs/jest-reporter.js:6:15 - error TS6133: 'globalConfig' is declared but its value is never read.

6   constructor(globalConfig, options) {
                ~~~~~~~~~~~~

libs/jest-reporter.js:29:69 - error TS2304: Cannot find name 'globalConfig'.

29       if (!testResult.console || testResult.console.length === 0 && globalConfig.verbose) {
                                                                       ~~~~~~~~~~~~


Found 2 errors.
```

File: e2e/package.json [hunk 1]
```diff
@@ -29,8 +29,8 @@
     "test-staging-nightly:debug": "cross-env WORKBENCH_ENV=staging PUPPETEER_DEBUG=true PUPPETEER_HEADLESS=false TEST_MODE=nightly-integration yarn _test --maxWorkers=1",
     "clean": "rimraf tsc-out",
     "build": "yarn clean && tsc -p tsconfig.json",
-    "lint": "eslint . --ext .js,.ts",
-    "lint:fix": "yarn lint --fix",
+    "lint": "eslint . --ext .js,.ts --quiet",
+    "lint:fix": "yarn lint --quiet --fix",
```
[the remark this answers is not in the document]
Comment (calbach) [reply e2e/package.json:33]: FYI here are my proposed changes to the e2e linter: https://precisionmedicineinitiative.atlassian.net/browse/RW-7698

**Review by calbach — COMMENTED** [review]
+1, this would be very useful

**Comment by aweng98** [comment]
Re: e2e linter. This is my (very close to done) [PR](https://github.com/all-of-us/workbench/pull/6407). Unsure what else is needed now because it needs rebase.

**Pull request merged**